php/WP_CLI/Dispatcher/Subcommand.php

```diff
@@ -480,7 +480,7 @@ public function invoke( $args, $assoc_args, $extra_args ) {
 				sprintf(
 					'wp %s %s',
 					$cmd,
-					ltrim( Utils\assoc_args_to_str( $actual_args ), ' ' )
+					ltrim( implode( ' ', [ Utils\args_to_str( $args ), Utils\assoc_args_to_str( $actual_args ) ] ), ' ' )
```

Comment:
```suggestion
					ltrim( implode( '', [ Utils\args_to_str( $args ), Utils\assoc_args_to_str( $actual_args ) ] ), ' ' )
```

This might fix the double space issue. `assoc_args_to_str()` will have a leading space, so no need to add another.

Reply:
I came up with a solution - forgot to push it before lunch :see_no_evil: - I hope this works now and passes all the tests.